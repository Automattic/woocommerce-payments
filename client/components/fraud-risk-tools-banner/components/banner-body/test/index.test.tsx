/**
 * External dependencies
 */
import React from 'react';
import { render } from '@testing-library/react';

/**
 * Internal dependencies
 */
import BannerBody from '..';

describe( 'BannerBody', () => {
	it( 'renders', () => {
		const { container: bannerBodyComponent } = render( <BannerBody /> );

		expect( bannerBodyComponent ).toMatchSnapshot();
	} );
} );
