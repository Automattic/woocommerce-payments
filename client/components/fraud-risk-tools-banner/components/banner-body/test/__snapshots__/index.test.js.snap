// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerBody renders 1`] = `
<div>
  <p
    class="discoverability-card__body"
  >
    A new range of tools have been added to WooCommerce Payments to help 
    <strong>
      reduce fraudulent transactions
    </strong>
     on your store. In addition to purchase verification, new risk filters are now available to screen incoming transactions - better protecting your store from fraudsters.
  </p>
</div>
`;
